repos/system_upgrade/el7toel8/actors/bindupdate/libraries/updates.py

```diff
@@ -0,0 +1,85 @@
+from leapp.libraries.common import isccfg
+from leapp.libraries.stdlib import run, api
+
+# Callback for walk function
+callbacks = {
+    'dnssec-lookaside': isccfg.ModifyState.callback_comment_out,
+}
+
+
+def paths_from_issues(issues):
+    """Extract paths from list of BindConfigIssuesModel."""
+    paths = []
+    for issue in issues:
+        paths.append(issue.path)
+    return paths
+
+
+def parser_file(parser, path):
+    for cfg in parser.FILES_TO_CHECK:
+        if cfg.path == path:
+            return cfg
+    return None
```

Comment:
Nit: return next((cfg for cfg in parser.FILES_TO_CHECK if cfg.path == path), None)

Reply:
to be honest, I like the original way more than the proposed oneliner :) but it's matter of taste. both are good

Reply:
Unless it is functionally different from my version, I would like keep it as it is. Seems more readable to me.